client/wfnews-war/src/main/angular/src/app/components/report-of-fire/photo-page/rof-photo-page.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, ChangeDetectionStrategy, ChangeDetectorRef, ElementRef } fro
 import { RoFPage } from "../rofPage";
 import { ReportOfFire } from "../reportOfFireModel";
 import { Camera, CameraResultType, CameraSource } from '@capacitor/camera';
+import { CommonUtilityService } from "../../../../../src/app/services/common-utility.service";
```

Comment:
I think it's time we start using absolute paths for this kind of thing, to hopefully prevent some build errors.

Add a section to the tsconfig for paths

```
"paths": {
    "@services/*": ["./src/app/services/*"]
  }
```
Or whatever the path is. Then you can use an import like:
`import { CommonUtilityService } from '@services'`

Reply:
ok, done!